dockstore-webservice/src/main/java/io/dockstore/webservice/resources/WorkflowResource.java

```diff
@@ -2139,12 +2139,25 @@ public void handleGitHubRelease(@ApiParam(hidden = true) @Parameter(hidden = tru
     public Response handleGitHubInstallation(@ApiParam(hidden = true) @Parameter(hidden = true, name = "user") @Auth User user,
             @Parameter(name = "X-GitHub-Delivery", in = ParameterIn.HEADER, description = "A GUID to identify the GitHub webhook delivery", required = true) @HeaderParam(value = "X-GitHub-Delivery")  String deliveryId,
             @RequestBody(description = "GitHub App repository installation event payload", required = true) InstallationRepositoriesPayload payload) {
+        final String action = payload.getAction();
+        // Currently, the action can be either "added" or "removed".
+        // This check is not necessary, but will detect if github adds another type of action to the event.
+        if (!List.of("added", "removed").contains(action)) {
```

Comment:
Share an enum with `dockstore-integration-testing/src/test/java/io/dockstore/webservice/helpers/GitHubAppHelper.java` so that if one is changed, the future me knows to change the other at the same time

Reply:
And link to `installation_repositories` I suppose